test/parallel/test-cluster-bind-twice.js

```diff
@@ -68,8 +68,9 @@ if (!id) {
 
   a.on('message', common.mustCall((m) => {
     if (typeof m === 'object') return;
-    assert.strictEqual(m, 'READY');
-    b.send('START');
+    assert(m.startsWith('READY '), 'message should start with "READY "');
+    const port = m.substr(6);
```

Comment:
I agree. I think even better is separating the message data from the port data by sending an object rather than a string. So I made it an object. Works locally, running CI in a moment. PTAL. Thanks for the review.